radix-engine/src/transaction/system_structure.rs

```diff
@@ -34,17 +34,14 @@ pub struct SystemFieldStructure {
 //       for the toolkit which parses it opaquely at the moment when doing transaction preview
 //       in the wallet. This change doesn't apply because BootLoader substates are only edited
 //       during transaction preview.
+//
+// Node depends on this structure, so append only.
 #[derive(Debug, Clone, ScryptoSbor, PartialEq, Eq)]
 pub enum SystemFieldKind {
     TypeInfo,
-    BootLoader(BootLoaderFieldKind),
```

Comment:
The enum used to be defined as:

```rs
#[derive(Debug, Clone, ScryptoSbor, PartialEq, Eq)]
pub enum SystemFieldKind {
    TypeInfo,
    BootLoader,
}
```

If we want to backward compat, shouldn't we revert to that?

Reply:
Yep, I did check this on `anemone`. 

Node only requires encoding and decoding, so it's backward compatible. :-)

Reply:
So you're saying that `SystemFieldKind::BootLoader` was never actually persisted and therefore removing that variant is not a breaking change?

Reply:
No, I essentially renamed `BootLoader` to `VMBoot` (as now we have more boot loader variants). 

I also appended two new variants.

Both should not affect the encoding and decoding of `BootLoaderField::BootLoader` (now called `BootLoader::VMBoot`).